docs/nzxt-smart-device-v1-guide.md

```diff
@@ -32,25 +37,25 @@ The device can report fan information for each channel, the noise level at the o
 ```
 # liquidctl status
 NZXT Smart Device (V1)
-├── Fan 1 speed                 1473  rpm
-├── Fan 1 voltage              11.91  V
-├── Fan 1 current               0.01  A
-├── Fan 1 control mode           PWM
-├── Fan 2 speed                 1341  rpm
-├── Fan 2 voltage              11.91  V
-├── Fan 2 current               0.02  A
-├── Fan 2 control mode            DC
-├── Fan 3 speed                 1352  rpm
-├── Fan 3 voltage              11.91  V
-├── Fan 3 current               0.02  A
-├── Fan 3 control mode           N/A
-├── Firmware version           1.0.7
-├── LED accessories                2
-├── LED accessory type    HUE+ Strip
-├── LED count (total)             20
-└── Noise level                   65  dB
+├── Fan 1 speed            1492  rpm
+├── Fan 1 voltage         11.91  V
+├── Fan 1 current          0.02  A
+├── Fan 1 control mode      PWM  
+├── Fan 2 speed            1368  rpm
+├── Fan 2 voltage         11.91  V
+├── Fan 2 current          0.02  A
+├── Fan 2 control mode      PWM  
+├── Fan 3 speed            1665  rpm
+├── Fan 3 voltage         11.91  V
+├── Fan 3 current          0.06  A
+├── Fan 3 control mode      PWM  
+└── Noise level              59  dB
 ```
 
+_The noise level is not available when data is read from [Linux hwmon]._
```

Comment:
That attribute is not available on hwmon drivers because there's no standard hwmon interface for that type of information.

But, and maybe more importantly, the noise is already missing (=always zero) on some devices, depending on the firmware version, and even on the devices that still report it, it's also not very useful (unknown but clearly poor resolution, unknown filter/weighting).  Plus the sensor is inside the case, so it has nothing to do with what noise reaches the user's ears.

Reply:
I meant more from a perspective of 'this used to show up and my external library that I made depends on there being x values in the array now one might sometimes be missing'. I think we have talked about it before, but is that something we are trying to avoid breaking?